src/workerd/api/crypto/rsa.c++

```diff
@@ -260,6 +478,13 @@ void Rsa::validateRsaParams(jsg::Lock& js,
   }
 }
 
+bool Rsa::isRSAPrivateKey(kj::ArrayPtr<const kj::byte> keyData) {
```

Comment:
Please add a comment pointing to the specification for this

Reply:
I'm not sure where the spec actually is. Pulled this from Node.js and implemented it to match node's behavior.